spec/services/push_notifications/event_notifier/post_logic_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+RSpec.describe ::PushNotifications::EventNotifier::PostLogic do
+  let(:user) { Fabricate.build(:user) }
+  let(:organization) { Fabricate(:organization) }
+  let(:post) { Fabricate(:post, organization: organization, user: user) }
+  let(:event) { Fabricate.build(:event, post: post, action: :created) }
+
+  describe '#users' do
+    context 'when a user pertains to the Post\'s organization' do
+
+      before { organization.members.create(user: user) }
+
+      context 'and the user has a DeviceToken associated' do
+
+        before { Fabricate(:device_token, user: user, token: 'aloha') }
```

Comment:
the spacing up until here makes it a bit hard to read. I'd simply keep the before's right after their context.